qml/plugins/CleanLocalsearch.qml

```diff
@@ -3,22 +3,21 @@ import Sailfish.Silica 1.0
 import Sailfish.Utilities 1.0
 
 ActionItem {
-    //% "Tracker database"
-    title: qsTrId("sailfish-tools-he-tracker_database")
+    //% "Localsearch database"
+    title: qsTrId("sailfish-tools-he-localsearch_database")
     //% "Clean"
     actionName: qsTrId("sailfish-tools-bt-clean")
     //: Text surrounded by %1 and %2 becomes a hyperlink: %1 is replaced by <a href="..."> and %2 by </a>.
-    //% "Tracker dabatabase cleaning can help in cases with "
-    //% "missing images, audio files etc. Processes using tracker "
-    //% "will be closed, tracker reindexing will be started. "
+    //% "Localsearch (formerly known as Tracker) dabatabase cleaning can help in cases with "
```

Comment:
Oh shame you were right next to 'dabatabase' and didn't catch it! That's been there for a very long time.